apps/dashboard/modules/extract/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { ptBR } from "date-fns/locale";
+import { Card } from "@repo/ui/card";
+import useSWR from "swr";
+import { format } from "date-fns";
+import { Box, Stack, Typography } from "@mui/material";
+import { Button } from "@repo/ui/button";
+import { useRouter } from "next/navigation";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+
+export function Extract() {
+  const { push } = useRouter();
+  const { data, isLoading, error } = useSWR("/api/extrato", fetcher);
+
+  if (isLoading) return <Card type="default">Carregando...</Card>;
+  if (error) return <Card type="default">Erro ao carregar</Card>;
```

Comment:
```suggestion
  if (isLoading) return <Card type="default" sx={{ width: "282px" }}>Carregando...</Card>;
  if (error) return <Card type="default" sx={{ width: "282px" }}>Erro ao carregar</Card>;
```

No carregamento está dando uma leve "piscada" por conta da largura do componente que não está padronizada. É só para melhorar a experiência mesmo.